qcfractal/storage_sockets/sqlalchemy_socket.py

```diff
@@ -1169,12 +1169,19 @@ def get_procedures(self,
             className = OptimizationProcedureORM
         elif procedure == 'torsiondrive':
             className = TorsionDriveProcedureORM
+        elif procedure == 'gridoptimization':
+            className = GridOptimizationProcedureORM
         else:
             # raise TypeError('Unsupported procedure type {}. Id: {}, task_id: {}'
             #                 .format(procedure, id, task_id))
-            # self.logger.warning('Procedure type not specified({}). Query include: Id= {}, task_id= {}'
-            #                 .format(procedure, id, task_id))
+            self.logger.warning('Procedure type not specified({}), so only ID is allowed. '
```

Comment:
Can we move this warning to below? If the `id` is specified this is a perfectly valid query. In addition, we might not want to print the id/task_id as this list could be very long.

Reply:
Ok, done